.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+on: [push,pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Add Credentials
+      env:
+        gh_token: ${{ secrets.GH_TOKEN }}
+      run: |
+        sed -i "s~''~'#'~g" bugswarm/common/credentials.py
+        sed -i "s~\[\]~\['${gh_token}'\]~g" bugswarm/common/credentials.py
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Is it possible to install `pip3` since were using `python version 3.6`?

Reply:
@octaviocorona Since we only have `python 3.6` installed the pip path is just for the specific python version.